components/FlexibleHeader/src/MDCFlexibleHeaderView.m

```diff
@@ -662,24 +663,29 @@ - (void)fhv_accumulatorDidChange {
   CGRect frame = self.frame;
 
   CGFloat frameBottomEdge = [self fhv_projectedHeaderBottomEdge];
-  frameBottomEdge = MAX(0, MIN(kShadowScaleLength, frameBottomEdge));
+  frameBottomEdge = MDCFMax(0, MDCFMin(kShadowScaleLength, frameBottomEdge));
 
-  CGFloat boundedAccumulator = MIN([self fhv_accumulatorMax], _shiftAccumulator);
+  CGFloat boundedAccumulator = MDCFMin([self fhv_accumulatorMax], _shiftAccumulator);
 
   CGFloat shadowIntensity;
   if (self.hidesStatusBarWhenCollapsed) {
     // Calculate the desired shadow strength for the offset & accumulator and then take the
     // weakest strength.
-    CGFloat accumulator = MAX(0, MIN(kShadowScaleLength, _minimumHeight - boundedAccumulator));
+    CGFloat accumulator = MDCFMax(0,
+                                  MDCFMin(kShadowScaleLength, _minimumHeight - boundedAccumulator)
+                                  );
     if (self.isInFrontOfInfiniteContent) {
       // When in front of infinite content we only care to hide the shadow when our header is
       // off-screen.
-      shadowIntensity = MAX(0, MIN(1, accumulator / kShadowScaleLength));
+      shadowIntensity = MDCFMax(0, MDCFMin(1, accumulator / kShadowScaleLength));
 
     } else {
       // When over non-infinite content we also want to hide the shadow when we're anchored to the
       // top of our content.
-      shadowIntensity = MAX(0, MIN(1, MIN(accumulator, frameBottomEdge) / kShadowScaleLength));
+      shadowIntensity = MDCFMax(0,
```

Comment:
If you add a linebreak after shadowIntensity will everything fit on the next line?